NLS.Adv.72.1.1/Highland_Pedigrees/kindred_34.trig

```diff
@@ -0,0 +1,214 @@
+
+<#EoinAhIle>
+    a foaf:Person;
+    irishRel:nomName "Eoin a hIle";
+    irishRel:ancestorOfGroup <#ClannDomnaill>, <#ClannRaghnaill>, <#ClannGhofraigh>.
+
+<#ClannDomnaill>
+    a irishRel:populationGroup;
+    irishRel:populationGroupName "Clann Domnaill".
+
+<#ClannRaghnaill>
+    a irishRel:populationGroup;
+    irishRel:populationGroupName "Clann Raghnaill".
+
+<#ClannGhofraigh>
+    a irishRel:populationGroup;
+    irishRel:populationGroupName "Clann Ghofraigh".
+
+<#Ailin>
+    a foaf:Person;
+    irishRel:nomName "Ailin".
+
+<#Eoin>
+    a foaf:Person;
+    irishRel:nomName "Eoin";
+    rdfs:comment "do bi dall fadeoigh".
+
+<#Domnall>
+    a foaf:Person;
+    irishRel:nomName "Domnall".
+
+<#AengusRiabach>
+    a foaf:Person;
+    irishRel:nomName "Aengus Riabhach".
+
+<#Dubghall>
+    a foaf:Person;
+    irishRel:nomName "Dubghall".
+
+<#Ruaidri>
+    a foaf:Person;
+    irishRel:nomName "Ruaidri";
+    rel:childOf <#Ailin>.
+
+<#Uisdiun>
+    a foaf:Person;
+    irishRel:nomName "Uisdiun";
+    rel:childOf <#Ailin>.
+
+<#Eoin-4ced7b60>
+    a foaf:Person;
+    irishRel:nomName "Eoin";
+    rel:childOf <#Ailin>.
+
+<#Domnaill>
+    a foaf:Person;
+    irishRel:genName "Domnaill";
+    irishRel:nomName "Domnall";
+    rel:childOf <#Raghnaill>;
+    owl:sameAs <http://example.com/NLS.Adv.72.1.1/Highland_Pedigrees/kindred_33.trig#Domnaill>.
+
+<#Raghnaill>
+    a foaf:Person;
+    irishRel:genName "Raghnaill";
+    irishRel:nomName "Raghnall";
+    owl:sameAs <http://example.com/NLS.Adv.72.1.1/Highland_Pedigrees/kindred_31.trig#Raghnaill>.
+
+<#Eoin-072fd310>
+    a foaf:Person;
+    irishRel:nomName "Eoin";
+    rel:childOf <#Domnaill>, <#Lainglib>.
+
+<#Lainglib>
+    a foaf:Person;
+    foaf:gender "female";
+    irishRel:nomName "Lainglib";
+    rel:childOf <#Cimair>.
+
+<#Cimair>
+    a foaf:Person;
+    irishRel:genName "Cimair";
+    irishRel:nomName "Mac Cimair".
+
+<#AlasdairNaCaille>
+    a foaf:Person;
+    irishRel:nomName "Alasdair na Caille";
+    rel:childOf <#Domnaill>.
+
+<#AengusOghar>
+    a foaf:Person;
+    irishRel:nomName "Aenghus Oghar";
+    rel:childOf <#Domnaill>.
+
+<#EoinDall>
+    a foaf:Person;
+    irishRel:nomName "Eoin Dall";
+    owl:sameAs <#Eoin>, <#Eoin-072fd310>;
+    rel:childOf [
+      a foaf:Person;
+      foaf:gender "female";
+      rel:childOf <#MhicCimi>;
+      owl:sameAs <#Lainglib>
+    ].
+
+<#MhicCimi>
+    a foaf:Person;
+    irishRel:genName "Mhic Cimi";
+    owl:sameAs <#Cimair>.
+
+<#Eoin-04ec2f30>
+    a foaf:Person;
+    irishRel:nomName "Eoin";
+    rel:childOf <#EoinDall>.
+
+<#AengusRiabhach>
+    a foaf:Person;
+    irishRel:nomName "Aengus Riabhach";
+    owl:sameAs <#AengusRiabach>;
+    rel:parentOf <#AenghusOg>;
+    rdfs:comment "aen mhac maith aige";
+    rdfs:seeAlso <https://www.wikidata.org/entity/Q76060800>.
+    #Black states that this individual died in 1440, but the Wikidata entry has
+    #him dying in 1400. Is this the same person? - EPT
```

Comment:
Black's article gives them an obit of 1440 while the Wikidata entry for the individual who seems to be the same person (based on genealogical relationships) has 1400.

Reply:
Ah right, I misread the comment and I saw 1440 twice rather than 1440 and 1400.  I would probably think that the wikidata is incorrect and Black is correct.